src/components/ha-combo-box-item.ts

```diff
@@ -0,0 +1,41 @@
+import { css } from "lit";
+import { customElement } from "lit/decorators";
+import { HaMdListItem } from "./ha-md-list-item";
+
+@customElement("ha-combo-box-item")
+export class HaComboBoxItem extends HaMdListItem {
+  static override styles = [
+    ...super.styles,
+    css`
+      :host {
+        --md-list-item-two-line-container-height: 64px;
+      }
+      md-item {
+        border-bottom: 1px solid var(--divider-color);
```

Comment:
We also show a line under the last item now, should probably exclude that one?

![CleanShot 2025-04-17 at 13 17 39@2x](https://github.com/user-attachments/assets/a566affe-cb60-4716-852b-9fe8fd80fda7)